src/lib.rs

```diff
@@ -1196,17 +1196,33 @@ pub fn anyref_heap_live_count() -> u32 {
 pub trait UnwrapThrowExt<T>: Sized {
     /// Unwrap this `Option` or `Result`, but instead of panicking on failure,
     /// throw an exception to JavaScript.
+    #[cfg(all(feature = "std", debug_assertions))]
+    #[track_caller]
+    fn unwrap_throw(self) -> T {
+        let loc = core::panic::Location::caller();
+        let msg = std::format!(
+            "`unwrap_throw` failed ({}:{}:{})",
+            loc.file(),
+            loc.line(),
+            loc.column()
+        );
+        self.expect_throw(&msg)
+    }
+
+    #[cfg(not(all(feature = "std", debug_assertions)))]
     fn unwrap_throw(self) -> T {
         self.expect_throw("`unwrap_throw` failed")
     }
 
     /// Unwrap this container's `T` value, or throw an error to JS with the
     /// given message if the `T` value is unavailable (e.g. an `Option<T>` is
     /// `None`).
+    #[track_caller]
```

Comment:
Could this (and the others below) be `#[cfg_attr(debug_assertions, track_caller)]`?